pyseir/deployment/model_to_observed_shim.py

```diff
@@ -0,0 +1,54 @@
+from typing import Sequence
+
+import numpy as np
+
+
+def shim_deaths_model_to_observations(
```

Comment:
Small comment from the PR grave - it would be great to throw in a small unit test for this function.

Going to be looking for opportunities to add slightly more testing to make refactoring easier in the future/examples of how the current functions are used.